web/js/codeworld.js

```diff
@@ -675,6 +675,15 @@ function help() {
         allowEscapeKey: true,
         allowOutsideClick: true,
         showConfirmButton: false,
+    }, function(confirmed){
+        // Kind of onClose event for sweetalert. Function get single bool argument -
+        // true if confirmed, false if not.
+        if (confirmed){
+            console.log("Should never happen. Doc sweetalert have no confirm option.")
```

Comment:
I don't think we'd care if this did happen, such as in a future redesign of the UI.  So how about not bothering to check, and just storing the position anyway?

Reply:
Done